### DISCUSSION TITLE
cmi playtime placeholders

### DISCUSSION THREAD

**Discussion by dan28000** [Zrips/CMI#7722] [Feature Requests]
have cplaytime placeholders - /cplaytime
current playtime placeholders are from stats of player

**Comment by 4lve** [comment]
%cmi_user_playtime_formatted% – Formatted playtime
%cmi_user_playtime_days% – Playtime in days
%cmi_user_playtime_dayst% – Playtime in days with fraction
%cmi_user_playtime_hours% – Playtime in hours
%cmi_user_playtime_hoursf% – Total playtime in hours
%cmi_user_playtime_hourst% – Total playtime in hours with fraction
%cmi_user_playtime_minutes% – Playtime in minutes
%cmi_user_playtime_minutest% – Total playtime in minutes
%cmi_user_playtime_seconds% – Playtime in minutes
%cmi_user_playtime_secondst% – Total playtime in minutes

**Comment by dan28000** [comment]
Those are taken from minecraft statistics
I'm asking for placeholders from CMI own cPlaytime ( /cmi cplaytime ) measurements

**Comment by CloudeLecaw** [comment]
Pls move it to discussions :)
